src/js/bodyparser.js

```diff
@@ -0,0 +1,40 @@
+// Module to add body parsing to express.
+"use strict";
+var namespace  = "gpii.express.middleware.bodyparser";
+var fluid      = fluid || require('infusion');
+var gpii       = fluid.registerNamespace("gpii");
+var bodyParser = fluid.registerNamespace(namespace);
+var bp         = require("body-parser");
+
+bodyParser.json = function(that, req, res, next) {
+    if (!that.privateJson) {
+        that.privateJson       = bp.json();
+    }
+
+    that.privateJson(req, res, next);
+};
+
+bodyParser.urlencoded = function(that, req, res, next) {
+    if (!that.privateUrlencoded) {
+        that.privateUrlencoded = bp.urlencoded({ extended: false });
+    }
+
+    that.privateUrlencoded(req, res, next);
+};
+
+fluid.defaults(namespace, {
```

Comment:
As above, use the string form for the namespace here - otherwise the arguments to fluid.defaults are not equivalent to their direct representation in JSON form.